go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	gopkg.in/gomail.v2 v2.0.0-20160411212932-81ebce5c23df
 	gorm.io/gorm v1.24.5
-	k8s.io/apimachinery v0.24.4
+	k8s.io/apimachinery v0.25.1
```

Comment:
Why are we bumping this ?

Reply:
Not sure, how this got bumped. Will revert it.